File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Enter Height: ");
    }
    while (n < 1 || n > 8);
    // This loop is to iterate rows
    for (int i = 1; i <= n; i++)
    {
        // this loop is to iterate dots and Print dots
        for (int j = 1; j <= n - i; j++)
        {
            printf(" ");
        }
        // This loop is to iterate hashes and Printing hashes
        for (int k = 1; k <= i; k++)
        {
            printf("#");
        }
        // This is for double spaces
        printf("  ");
        // this is to print hashes of right aligned pyramid
        for (int l = 1; l <= i; l++)
        {
            printf("#");
        }
        // this is to print hashes of left aligned pyramid

        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

height = 0

# check for height to be between 1 and 8
while height < 1 or height > 8:
    height = get_int("Enter Height: ")
    # This loop is to iterate rows
for i in range(1, height + 1):
    # This loop is to get spaces before the hashes
    for j in range(height - i):
        print(" ", end="")
    # This loop is to iterate hashes and Printing hashes of left side of pyramid
    for k in range(i):
        print("#", end="")
    #  Printing two whitespaces between sides of pyramid
    print("  ", end="")
    for l in range(i):
        # This loop is to iterate hashes and Printing hashes of right side of pyramid
        print("#", end="")
    print()
